app/views/posts/edit.html.slim

```diff
@@ -0,0 +1 @@
+= render partial: "posts/shared/form", locals: { post: @post }
```

Comment:
Priekš kam tev shared folderis?
Varetu iemest to iekš posts un rails paņemtu to pēc relativa path.:

= render "form", locals: { post: @post }

Reply:
ja zinu ka ta , bet man kka organizatoriskak liekas ielikt tadu failu , tada mapite.

Reply:
Nu skaties, shared, nozīme kā foldera saturs tiks izmantots kaut kur vairakkārt projektā. Tā kā shared behavior.
Lielos projektos iekš shared parasti: modal, table, havbar un t.t.

Bet tev tagad shared ir zem posts, un sanak kā tas ir shared behaviour only for posts - un it kā tas vairs nav shared behaviour at all.

Vel ir tada lieta, kā rails izmanto convention (tas ir rails fiška, convention over configuration), kā visi ne skati bet mazak elementi, kuri saucas par partials sakas ar underline un citiem developeriem kļust skaidrs kas tas ir.

Ar to viss ok, jā tev posts folderi būs: show.slim, edit.slim, _form.slim, _table.slim, _filter.slim un t.t.

Reply:
A , understood tagad , parliku uz vienkarsi posts folder